modules/registry-accessing-directly.adoc

```diff
@@ -22,7 +22,7 @@ $ oc get nodes
 +
 [source,terminal]
 ----
-$ oc debug nodes/<node_address>
+$ oc debug nodes/<node_name>
```

Comment:
Actually I just noticed that the step text above says "Access the node by getting the node's address". Should this be updated to instead say something about accessing the node by passing in the node name?

Reply:
+1